src/com/codefortomorrow/advanced/chapter16/solutions/LinkedList.java

```diff
@@ -0,0 +1,153 @@
+package com.codefortomorrow.advanced.chapter16.solutions;
+
+// UUID to represent each node's unique ID
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+// using Lombok for Getters and Setters (saves lines)
+import lombok.Setter;
+import lombok.Setter;
+
+/**
+ * @author ArmeetJatyani
+ * March 2021
+ *
+ * Implement a simple LinkedList
+ * You will need to create a LinkedListNode class, which represents each item/node in the linked list
+ * Required functionality...
+ * - head(): get head of list
+ * - tail(): get tail of list
+ * - add(): add to tail of list
+ * - push(): push to head of list
+ * - pop(): remove head of list
+ * - toString(): return a String representation of the list
+ */
+
+@Setter
+public class LinkedList {
+
+    private LinkedListNode head;
+
+    /**
```

Comment:
This might be a little tedious but for the sake of completeness could you please flesh out all of the method javadoc comments? This one is fine but for example the `LinkedList(int value)` javadoc comment doesn't have a parameter description for ` value`.

Reply:
These are auto-generated. If it's distracting, I can just remove the @param args. They're a bit redundant.